src/mmw/js/src/modeling/tests.js

```diff
@@ -294,26 +315,23 @@ describe('Modeling', function() {
                     assert.isTrue(scenario1Spy.calledOnce, 'Scenario1 was not saved once.');
                     assert.isTrue(scenario2Spy.calledOnce, 'Scenario2 was not saved once.');
                 });
+                 */
 
                 it('properly associates scenarios with the project when initially saving the project', function() {
                     var projectResponse = '{"id":21,"user":{"id":1,"username":"test","email":"test@azavea.com"},"name":"Test Project","area_of_interest":{},"is_private":true,"model_package":"tr-55","created_at":"2015-06-03T20:09:11.988948Z","modified_at":"2015-06-03T20:09:11.988988Z"}',
```

Comment:
I think @rajadain moved some of these long responses to a separate file to keep things tidy. Consider a similar approach.